contrib/buildinfo/src/BuildInfo.scala

```diff
@@ -15,33 +15,35 @@ trait BuildInfo extends ScalaModule {
     Map.empty[String, String]
   }
 
-  private def generateBuildInfo(members: Map[String, Any])(implicit dest: Ctx.Dest, log: Ctx.Log): Seq[PathRef] =
-    if (!members.isEmpty) {
-      val outputFile = dest.dest / "BuildInfo.scala"
+  def generatedBuildInfo: T[Seq[PathRef]] = T {
+    val logger: Logger = T.ctx.log
```

Comment:
According to the documentation, it should be `T.ctx().log`.

Reply:
done